src/interfaces/stepper-interfaces/DefineCompany/define-company-interface.ts

```diff
@@ -0,0 +1,6 @@
+import { BusinessContactIF } from './business-contact-interface'
+
+export interface DefineCompanyIF {
+    defineCompanyStepValid: boolean
```

Comment:
I suggest renaming this since we already know we're in the Define Company IF, and also it would be nice if all step IFs had the same "valid" property name.

Reply:
Thats a great suggestion. Will rename it so that all steps has the property 'valid'